manifests/plugin/snmp_agent.pp

```diff
@@ -44,10 +44,7 @@
 
   collectd::plugin { 'snmp_agent':
     ensure  => $ensure,
-    content => epp('collectd/plugin/snmp_agent.conf.epp', {
-      'data'  => $data,
```

Comment:
I don't quite understand how this makes a difference. The variable `$data` and also `$table` are available in this context. Can you please provide an acceptance test for this?

Reply:
Sure ill write some additional tests. As for how this makes a difference im not entirely sure I understand it myself to be honest. As far as I can make sense of it taking into account the error im getting. It looks like when it tries to get the $data out of this context it ends up being null for some reason and although data and table are optional in the template it break. So I tried to work around this by removing this part and getting the variables by a full path in the template. This seems to work correctly both on tripleo and the existing tests on puppet.